luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/factories/PermissivePhysicalTableFactory.java

```diff
@@ -0,0 +1,36 @@
+// Copyright 2019 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.config.luthier.factories;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.yahoo.bard.webservice.config.luthier.LuthierIndustrialPark;
+import com.yahoo.bard.webservice.data.config.LuthierPhysicalTableParam;
+import com.yahoo.bard.webservice.table.ConfigPhysicalTable;
+import com.yahoo.bard.webservice.table.PermissivePhysicalTable;
+
+/**
+ * A factory that is used by default to support Simple (non-Composite) Physical Table.
```

Comment:
We should specify here what it means for a physical table to be "permissive."

Reply:
Here or in the corresponding subclass? @archolewa

Reply:
This is the corresponding subclass.

Reply:
Oh you, mean here or `PermissivePhysicalTable`. I don't care.

Reply:
I wrote them in the corresponding subclass and hinted them here. 
But I'd like to talk with @michael-mclawhorn before pushing.